modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/coldstorage/MoveToColdStorage.java

```diff
@@ -67,4 +73,17 @@ public DocumentModel run(DocumentModel doc) {
         return documentModel;
     }
 
+    @OperationMethod
+    public DocumentModelList run(DocumentModelList documents) {
+        DocumentModelList result = new DocumentModelListImpl();
+        for (DocumentModel documentModel : documents) {
+            try {
+                result.add(run(documentModel));
+            } catch (Exception e) {
+                log.error("Unable to move document: {} to cold storage", documentModel.getId(), e);
+            }
```

Comment:
If we confirm that we want this kind of pattern, should we also add a test for it?